ocdaction/ocdaction/settings/__init__.py

```diff
@@ -31,7 +31,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'widget_tweaks',
+#    'widget_tweaks',
```

Comment:
can we comment this back in? was it causing issues? The forms are using this now